src/modules/load_mon/load_mon.cpp

```diff
@@ -58,6 +58,10 @@
 #include <uORB/topics/cpuload.h>
 #include <uORB/topics/task_stack_info.h>
 
+#if !defined(CONFIG_SCHED_INSTRUMENTATION)
```

Comment:
Without having full context...

I see #ifdef __PX4_NUTTX in the file. This implies that it may be built on non-nuttx builds. If that is true then CONFIG_SCHED_INSTRUMENTATION is a nuttx based define and should be 1) located in nuttx code or be conditional on __PX4_NUTTX

Reply:
Good point. I assumed that `CONFIG_SCHED_INSTRUMENTATION` will only be defined if built with NuttX anyway.